### PR TITLE
Increases the cost of HPR/SVD to 300.

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Title literally.

But if you *really* need to know.

SVD 150->300
HPR 150->300
Magazines unaffected.

## Why It's Good For The Game
HPR and SVD are killer deals for 150. With both sharing ammo with common street rifles that marines get ammo boxes for (Rifle!Family/Mosin respectively)
This should put them to a higher point value and cost and make it more obvious they're something inbetween a 40 point and the usual imports which either fall into 120 or 50 points respectively.

tl;dr They're way too good of killer deals at 150.

## Changelog

:cl:
balance: SVD and HPR now cost 300 points to buy in Requisitions instead of 150 to reflect their status.
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
Alternatively I had other thoughts like just nuking these from req. Both of them don't really fit in there as the HPR is basically an ERT gun in terms of gun quality, SVD is just mosin+ in basically every way. I preferred doing this though.

**Comment by SimplySmiley123** [comment]
It's already rare to see someone _order_ these from requisitions?
SVD especially considering you _can_ get it for free depending on the map and your luck.

**Comment by 00ze-cyclone** [comment]
> SVD especially considering you _can_ get it for free depending on the map and your luck.

Bro maps being retarded doesn’t justify making something cheaper, if anything you are just giving MJP more reasons to increase the price even further you fool.

**Comment by ivanmixo** [comment]
Uh could you just revert the chages that weird commit the bot did so the PR is good to go? @Pariah919

**Review by ivanmixo — APPROVED** [review]

**Comment by spookydonut** [comment]
You will have to make a new PR using a new branch in your fork that isn't master.
Your fork's master is running the changelog bot which is polluting this PR with changelog bot commits.
Make sure your new branch doesn't contain these erroneous changelog commits.
